packages/blockchain-ui/enzyme/tests/TutorialSinglePage.test.tsx

```diff
@@ -9,7 +10,7 @@ import ITutorialObject from '../../src/interfaces/ITutorialObject';
 chai.should();
 chai.use(sinonChai);
 
-describe('TutorialPage component', () => {
+describe.only('TutorialPage component', () => {
```

Comment:
need to remove the only. This should have made the tests fail for lack of coverage!?